src/extensions/extension-loader.ts

```diff
@@ -47,7 +46,7 @@ const logModule = "[EXTENSIONS-LOADER]";
  */
 export class ExtensionLoader extends Singleton {
   protected extensions = observable.map<LensExtensionId, InstalledExtension>();
-  protected instances = observable.map<LensExtensionId, LensExtension>();
+  instances = observable.map<LensExtensionId, LensExtension>();
```

Comment:
I don't think this should be public. A getter for the list of instances could be useful if it doesn't already exist.

Reply:
Agreed, this being exposed for external mutation is scary. Good catch.